app/models/foreman_openscap/compliance_status.rb

```diff
@@ -8,6 +8,10 @@ def self.status_name
       N_('Compliance')
     end
 
+    def status_link
+      host.arf_reports_path(:search => "host = #{host.id}")
```

Comment:
Should the search be `host = #{host.name}`?

Reply:
ir should it even link to the last report directly? note that status represents results of the last scan

Reply:
If host has multiple policies and several of them are failing, showing just one report might be misleading. I think we are better off showing a list of reports rather that a single one.